Documentation/Signoffs/LaunchingAndFiring.md

```diff
@@ -17,22 +17,26 @@ This subsystem will use a 12-volt DC brushed motor along with a relay that will
 
 **Constraint Solution 2:**
-To allow the firing mechanism to operate without human interaction, a signal shall be generated from the Aiming subsystem when that subsystem has finished the aiming processes that will be used as a relay signal to operate the firing mechanism autonomously. The relay[3] will disconnect the 12-volt supply from the DC motor when the signal is not active, or low, and will connect the 12-volt supply to the DC motor when the signal is activated, or high, allowing the motor to operate. The relay will be connected to pin 1 of the Raspberry Pi 5 from the aiming subsystem and will pull 100mA at max (while operating) which is within the Raspberry Pi 5's ampacity limits for that pin. The relay, the Adafruit Power Relay FeatherWing G5LE-G is rated for 10 Amp resistive loads at 12 Volts DC. This is over the motor's stall ampacity and it also useful to consider that the team will be operating the motor at much ampacity than this at all times however this relay can handle 10 amps at 12 volts regardless.
+To allow the firing mechanism to operate without human interaction, a signal shall be generated from the Aiming subsystem when that subsystem has finished the aiming processes that will be used as a relay signal to operate the firing mechanism autonomously. The relay[3] will disconnect the 12-volt supply from the DC motor when the signal is not active, or low, and will connect the 12-volt supply to the DC motor when the signal is activated, or high, allowing the motor to operate. The relay will be connected to pin 1 of the Raspberry Pi 5 from the aiming subsystem and will pull 100mA at max (while operating) which is within the Raspberry Pi 5's ampacity limits for that pin. The relay, the Adafruit Power Relay FeatherWing G5LE-G is rated for 10 Amp resistive loads at 12 Volts DC. This is over the motor's stall ampacity and it also useful to consider that the team will be operating the motor at much ampacity than this at all times however this relay can handle 10 amps at 12 volts regardless. After the shot is and preloading sequence is completed, a proximity sensor (mounted by ME team) will sense a bolt during that part of the rotation cycle and will send a signal to the Raspberry Pi 5 GPIO 11 (Pin23) to disengage the PMDC motor allowing the motor to come to rest against the resistance of the cam in the ME team's design at a predetermined point each time.
```

Comment:
Will the raspberry pi take a 12v input from the prox? This prox sensor is a photo prox not an inductive prox. Will it be able to "see" a metal bolt?